README.md

```diff
@@ -1,54 +1,314 @@
+
+For anything more complicated than building, you'll need an IDE like [VS Community 2017](https://visualstudio.microsoft.com/vs/community), currently the best free C# IDE. Open `BizHawk.sln` with VS to start and use the toolbar to choose EmuHawk and build. See [Compiling at TASVideos](http://tasvideos.org/Bizhawk/Compiling.html) (somewhat outdated) for more detailed instructions.
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### GNU+Linux and macOS
+
+*Compiling* requires MSBuild and *running* requires Mono and WINE, but **BizHawk does not run under WINE** — only the bundled libraries are required.
+
+If you use GNU+Linux, there might be a `bizhawk-git` package or similar in the same repo as the main package. If it's available, installing it will automate the build process.
+
+Building is as easy as:
+```sh
+git clone https://github.com/TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+# or ssh: git clone git@github.com:TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+msbuild /p:Configuration=Release BizHawk.sln
+```
+
+Remove the `/p:...` flag from MSBuild if you want debugging symbols.
+
+If your distro isn't listed under *Installing* above, `libblip_buf` probably isn't in your package repos. You can easily [build it yourself](https://gitlab.com/TASVideos/libblip_buf/blob/unified/README.md).
+
+Once built, see [*Installing*](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos) above, substituting the repo's `output` folder for the download.
+
+Again, if your distro isn't listed there, you might get an "Unknown distro" warning in the terminal, and BizHawk may not open or may show the missing dependencies dialog. You may need to add your distro to the case statement in the script, setting `libpath` to the location of `d3dx9_43.dll.so` (please do share if you get it working).
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Usage
+
+#### Loading firmware
+
+You may have seen a dialog saying "You are missing the needed firmware files [...]" when trying to open a rom. Pressing "Yes" opens the Firmware Manager, or you can go to `Config` > `Firmwares...`.
+
+To load firmwares, the easiest way is to click "Import" in the menubar, navigate to the dumped firmware(s), select them all, and click "Open". It's a good idea to have them copied into the `Firmware` folder, which is nicely organised, when prompted. If you were trying to open a rom, click "Close and reload ROM" to do that. Keep in mind some firmware is optional and some have multiple versions, only one of which needs to be set.
+
+#### Identifying a good rom
+
+With a core and game loaded, look in the very left of the status bar (`View` > `Display Status Bar`):
+* a green checkmark means you've loaded a "known good" rom;
+* a "???" (don't remember) means you've loaded a "known bad" rom, created by incorrect dumping methods; and
```

Comment:
🚫? ✗?

Reply:
https://github.com/TASVideos/BizHawk/blob/master/BizHawk.Client.EmuHawk/images/RetroQuestion.png

Reply:
https://github.com/TASVideos/BizHawk/blob/master/BizHawk.Client.EmuHawk/images/ExclamationRed.png

Reply:
oh there's more than three... better idea is "something else"